hw/ip/spi_host/doc/checklist.md

```diff
@@ -41,22 +41,22 @@ Code Quality  | [LINT_SETUP][]                 | Done        |
 Type          | Item                    | Resolution  | Note/Collaterals
 --------------|-------------------------|-------------|------------------
 Documentation | [NEW_FEATURES][]        | Done        |
-Documentation | [BLOCK_DIAGRAM][]       | Not Started |
-Documentation | [DOC_INTERFACE][]       | Not Started |
-Documentation | [MISSING_FUNC][]        | Not Started |
+Documentation | [BLOCK_DIAGRAM][]       | Done        |
+Documentation | [DOC_INTERFACE][]       | Done        |
+Documentation | [MISSING_FUNC][]        | Done        |
 Documentation | [FEATURE_FROZEN][]      | Done        |
 RTL           | [FEATURE_COMPLETE][]    | Done        |
-RTL           | [AREA_CHECK][]          | Not Started |
+RTL           | [AREA_CHECK][]          | Done        |
 RTL           | [PORT_FROZEN][]         | Done        |
 RTL           | [ARCHITECTURE_FROZEN][] | Done        |
-RTL           | [REVIEW_TODO][]         | Not Started |
-RTL           | [STYLE_X][]             | Not Started |
-Code Quality  | [LINT_PASS][]           | Not Started |
-Code Quality  | [CDC_SETUP][]           | Not Started |
-Code Quality  | [FPGA_TIMING][]         | Not Started |
+RTL           | [REVIEW_TODO][]         | Done        | 
```

Comment:
I did a quick search. There are still a bunch of TODOs in DV code - but that is OK.
On the design side I found this:
https://github.com/lowRISC/opentitan/blob/4c3ae9eaf131ebe2fb5e7f25f6430ca5bc15b3b5/hw/ip/spi_host/rtl/spi_host.sv#L9

Is this still relevant? Or can it be removed?

Reply:
Ah, yes, that is old. It as been implemented.  It can be removed.

Reply:
Done.